frontend/src/components/common/Resource/EditorDialog.tsx

```diff
@@ -109,34 +113,41 @@ export default function EditorDialog(props: EditorDialogProps) {
 
   const theme = useCurrentAppTheme();
 
-  const [useSimpleEditor, setUseSimpleEditorState] = React.useState(() => {
-    const localData = localStorage.getItem('useSimpleEditor');
-    return localData ? JSON.parse(localData) : false;
-  });
+  const [hideManagedFields, setHideManagedFields] = useLocalStorageState<boolean>(
+    'hideManagedFields',
+    false
```

Comment:
I'd make the default setting here "true", that's what kubectl does by default and I think we should too. But this is not too critical since you only have to check it once and it'll remember it.
What do you think @farodin91 @joaquimrocha ?

Reply:
Yeah, let's turn it on by default please.